### PR TITLE
social media icons added | issue #18 fixed

### LINKED ISSUE #18: feat: Add our social media icons
The section here is empty, maybe add social media icons of Twitter, LinkedIn, Instagram, GitHub.

image

You can found the socials link here:
https://bio.link/web3community

### PR DESCRIPTION
**Pull request by 0shivamh**
## Related Issue 
fixed #18 issue 
- Information about the related issue 

Closes: # 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 updated screenshot :
 
change

### CODE REVIEW

**Comment by 0shivamh** [comment]
HI @vinzvinci  please review it, if there are any changes please let me know

**Review by mkubdev — COMMENTED** [review]
Hi, good PR. But i think we should use [Tailwind](https://tailwindcss.com/) ..

**Review by vinzvinci — CHANGES_REQUESTED** [review]
Can you change it to tailwind? 

We like to follow that framework.

**Comment by 0shivamh** [comment]
> Can you change it to tailwind?
> 
> We like to follow that framework.

Okay, No issue

**Comment by vinzvinci** [comment]
Thank you :)

**Comment by 0shivamh** [comment]
> Thank you :)

Can I use .svg for the icon?  or should I continue with fontawesome? , If you have any other suggestions please let me know

**Comment by vinzvinci** [comment]
> Can I use .svg for the icon? or should I continue with fontawesome? , If you have any other suggestions please let me know

You can use fontawesome

**Comment by 0shivamh** [comment]
> > Can I use .svg for the icon? or should I continue with fontawesome? , If you have any other suggestions please let me know
> 
> You can use fontawesome

okay, thank you.

File: main.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    font-size: 40px;
+    position: relative;
+    top: 0;
+    transition: top ease 0.3s;
+    }
+.social-icons:hover{
+    color: gray;
+    top: -5px;
+    transition: all 0.15s;
+}
```
Comment (Panquesito7) [inline main.css:20]: Missing an endline.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.social-icons{
+    padding: 5px;
+    text-decoration: none;
+    color: white;
+    font-size: 40px;
+    position: relative;
+    top: 0;
+    transition: top ease 0.3s;
+    }
```
Comment (Panquesito7) [inline main.css:15]:
```suggestion
}

```

File: main.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+html {
+    scroll-behavior: smooth;
+    transition: 3s;
+}
+
+
```
Comment (Panquesito7) [inline main.css:6]:
```suggestion
```

File: index.html [hunk 4]
```diff
@@ -247,10 +249,20 @@ <h4 id="name" class="text-xl font-semibold mb-2">Zapier</h4>
       <div class="container mx-auto px-8">
         <div class="w-full flex flex-col md:flex-row py-6">
           <div class="flex-1 mb-6 text-black">
-            <a class="text-white no-underline hover:no-underline font-bold text-2xl lg:text-2xl" href="#">
+            <!-- <a class="text-white no-underline hover:no-underline font-bold text-2xl lg:text-2xl" href="#"> -->
               <!--Icon from: http://www.potlabicons.com/ -->
-              Social
-            </a>
+              <!-- Social -->
+            <!-- </a> -->
```
Comment (Panquesito7) [inline index.html:255]:
```suggestion
            <!-- Social -->
            <!-- </a> -->
```

File: main.css [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+.social-icons{
+    padding: 5px;
+    text-decoration: none;
+    color: white;
+    font-size: 40px;
+    position: relative;
+    top: 0;
+    transition: top ease 0.3s;
+    }
+.social-icons:hover{
```
Comment (Panquesito7) [inline main.css:16]:
```suggestion
.social-icons:hover {
```

File: main.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
+html {
+    scroll-behavior: smooth;
+    transition: 3s;
+}
+
+
+.social-icons{
```
Comment (Panquesito7) [inline main.css:7]:
```suggestion
.social-icons {
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Also, there's no screenshot provided. Could you please add some screenshots? Thank you. 🙂

**Comment by 0shivamh** [comment]
> Also, there's no screenshot provided. Could you please add some screenshots? Thank you.

sure

**Review by vinzvinci — APPROVED** [review]
LGTM 👍 We can come up with new ideas if needed.

For final review @mkubdev.

**Comment by mkubdev** [comment]
LGTM! 👍

File: index.html [hunk 7]
```diff
@@ -247,12 +248,26 @@ <h4 id="name" class="text-xl font-semibold mb-2">Zapier</h4>
...
+
+            <!-- social icons start -->
+            <p class="text-white md:mb-6">Follow Us</p>
```
Comment (Panquesito7) [inline index.html:257]:
```suggestion
            <p class="text-white md:mb-6">Follow us on social media</p>
```

File: index.html [hunk 8]
```diff
@@ -247,12 +248,26 @@ <h4 id="name" class="text-xl font-semibold mb-2">Zapier</h4>
...
+
+              <a aria-hidden='true' class='text-white fa fa-twitter fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100' href='https://twitter.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+
+              <a aria-hidden='true' class='text-white fa fa-linkedin fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100' href='https://www.linkedin.com/company/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+
+              <a aria-hidden='true' class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100' href='https://github.com/web3community' rel='noreferrer' target='_blank' title='follow us'></a>
+            </div>
 
-            <!--Put socials icons here-->
+             <!--socials icons ended-->
```
Comment (Panquesito7) [inline index.html:270]:
```suggestion
            <!--socials icons ended-->
```

**Review by Panquesito7 — COMMENTED** [review]
Woah! It looks awesome! Great work! 🎉

**Comment by 0shivamh** [comment]
> Woah! It looks awesome! Great work!

happy to hear that, thank you

**Comment by Panquesito7** [comment]
> LGTM! 👍

My reviews weren't resolved yet 😅 I'll make another PR to fix that.

**Comment by mkubdev** [comment]
Oops 🤭 sry mate.

**Comment by Panquesito7** [comment]
It's alright. I'll make another PR for that if you like it. 🙂

**Pull request merged**